app/templates/components/forms/session-speaker-form.hbs

```diff
@@ -129,12 +129,26 @@
       {{#if field.isIncluded}}
         <div class="field">
           <label class="{{if field.isRequired 'required'}}" for="name">{{field.name}}</label>
-          {{#if (is-input-field field.type) }}
-            {{#if field.isLongText}}
-              {{widgets/forms/rich-text-editor value=(mut (get data.speaker field.fieldIdentifier))
+          {{#if (eq field.name "Speaking Experience") }}
+            {{widgets/forms/rich-text-editor value=(mut (get data.speaker field.fieldIdentifier))
                 textareaId=(if field.isRequired (concat 'speaker_' field.fieldIdentifier '_required') (concat 'speaker_' field.fieldIdentifier))}}
+          {{else}}
+            {{#if (eq field.name "Gender") }}
+              <select  id="gender">
```

Comment:
use ember-semantic-ui components like in rest of the app.

Reply:
@CosmicCoder96  This issue needs to be solved from backend.